example/src/ConversationListView.tsx

```diff
@@ -18,6 +18,7 @@ export default function ConversationListView({
 
   async function refreshConversations() {
     const conversations = await client.conversations.list();
+    XMTPPush.subscribe(conversations.map((c) => c.topic));
```

Comment:
It only subscribes once you refresh the list view. Not super great at react couldn't figure out how to get it to subscribe on streaming conversations but this is a minor detail and we can improve it later.